README.md

```diff
@@ -240,6 +241,17 @@ const mapDispatchToProps = (dispatch, props) => {
 };
 ```
 
+The result of the promise returned by `mutateAsync` will be the following object:
+
+| Name | Type | Description |
+|:----|:-----|:-----|
+| status | number | HTTP status code
+| body | object | Response body auto-parsed by SuperAgent
+| text | string | Unparsed response body string
+| duration | number | Elapsed time between the action dispatched and resolved
+| transformed | any | Normalized data by the `transform` function
```

Comment:
Change to "Result from the `transform` function. Will be identical to `body` if `transform` is unprovided in the query config."